ios/PurchasesHybridCommon/PurchasesHybridCommonUI/CustomerCenter/RCCustomerCenterViewController.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  RCCustomerCenterViewController.swift
+//  Pods
+//
+//  Created by Facundo Menzella on 17/2/25.
+//
+
+#if !os(macOS) && !os(tvOS) && !os(watchOS)
+
+import Foundation
+import PurchasesHybridCommon
+import RevenueCat
+import RevenueCatUI
+import UIKit
+import SwiftUI
+
+@available(iOS 15.0, *)
+@available(macOS, unavailable)
+@available(tvOS, unavailable)
+@available(watchOS, unavailable)
+@objc
+class RCCustomerCenterViewController: UIViewController {
+
+    /// Create a view controller to handle common customer support tasks
+    /// - Parameters:
+    ///   - customerCenterActionHandler: An optional `CustomerCenterActionHandler` to handle actions
+    ///   from the Customer Center.
+    public init(
+        customerCenterActionHandler: CustomerCenterActionHandler? = nil
+    ) {
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    public override func viewDidLoad() {
+        super.viewDidLoad()
+
+        if self.hostingController == nil {
+            self.hostingController = self.createHostingController()
+        }
+    }
+
+    @available(*, unavailable, message: "Use init(customerCenterActionHandler:mode:) instead.")
+    required dynamic init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Private
+
+    private var hostingController: UIViewController? {
+        willSet {
+            guard let oldController = self.hostingController else { return }
+
+            oldController.willMove(toParent: nil)
+            oldController.view.removeFromSuperview()
+            oldController.removeFromParent()
+        }
+
+        didSet {
```

Comment:
Hmm I don't remember the reasons for these `willSet`/`didSet` changes with the view controllers 😅. It should probably be ok, but not sure if there is anything more we can remove now that we don't have to support footers here.

Reply:
I kept them for consistency, but now I am thinking that it makes things way more complicated.
I'll remove them, and do it inline